### ISSUE TITLE
WinApps-Launcher: ERROR: 'winapps' NOT FOUND.

### ISSUE THREAD

**Issue by onlineapps-cloud** [winapps-org/WinApps-Launcher#10]
hi, i have installed winapps, but launcher don't see it.
error:
```
ERROR: 'winapps' NOT FOUND.
Please ensure 'winapps' is installed.
```
```
./installer.sh
################################################################################
#                                                                              #
#                            WinApps Install Wizard                            #
#                                                                              #
################################################################################

Q) Install or uninstall WinApps? --> Install
Q) Configure WinApps for the current user 'user' or the whole system? --> Current User
Q) Automatically install supported applications or choose manually? --> Manual (Default)

Installing WinApps.
Checking for existing conflicting WinApps installations... Failed!

ERROR: EXISTING 'USER' WINAPPS INSTALLATION.
A previous WinApps installation was detected for the current user.
--------------------------------------------------------------------------------
Please remove the existing WinApps installation using ./installer.sh --user --uninstall.
--------------------------------------------------------------------------------
Exiting with status '3'.
```

**Comment by oskardotglobal** [comment]
Please ensure that `winapps` is on `PATH`.

**Comment by onlineapps-cloud** [comment]
app was not found in PATH, 
winapps was downloaded here` /home/user/Downloads/winapps-main/`
i added winapps in PATH with this command: `export PATH=/home/user/Downloads/winapps-main/bin/:$PATH`
right now if i type in console `winapps` i get this error:
![image](https://github.com/user-attachments/assets/78c9c43f-4b29-4022-888f-4794a483c506)
if i run `./WinAppsLauncher.sh` i get same error, what i do wrong?

**Comment by oskardotglobal** [comment]
You have to add `$HOME/.local/bin` to `$PATH`, not the path to your WinApps installation.
Sidenote: You should really move WinApps out of your Downloads folder

**Comment by oskardotglobal** [comment]
Your `$PATH` is still wrong

**Comment by onlineapps-cloud** [comment]
can you explain more complex, what i need exactly, im not familiarized with PATH, what folder, where to move, which command i need to execute, thanks a lot!

**Comment by oskardotglobal** [comment]
`PATH` is a variable that determines which commands the system will find. Add the following line to your `~/.bashrc`: `export PATH="$HOME/.local/bin:$PATH"`, then restart your Terminal.

**Comment by onlineapps-cloud** [comment]
i made `nano  ~/.bashrc`
and to the bottom of the file and add `export PATH="$HOME/.local/bin:$PATH"` and save
![image](https://github.com/user-attachments/assets/b38f6a6b-682e-47a8-bce3-6578f9793a4c)
i closed terminal, and opened it again, and checked

> echo $PATH
> /home/user/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/sbin:/usr/sbin:/app/lib/010editor

if i execute winapps, i get same error
```
Application not found.
Please ensure the program is correctly configured as an officially supported application. 
```
what i do wrong?

**Comment by onlineapps-cloud** [comment]
upd, i deleted winapps folder, and download again, uninstall, and install again, after these manipulations in Winapps launcher i have another error:
![image](https://github.com/user-attachments/assets/2c39596b-1925-49cf-8b56-79c020455d3f)

**Comment by onlineapps-cloud** [comment]
in config `/home/user/.config/winapps/winapps.conf` i configured as manual, because i connecting to remote machine
```
# [WINAPPS BACKEND]
# DEFAULT VALUE: 'docker'
# VALID VALUES:
# - 'docker'
# - 'podman'
# - 'libvirt'
# - 'manual'
WAFLAVOR="manual"
```

**Comment by oskardotglobal** [comment]
The launcher doesn't currently support systems of type `manual`.

**Comment by onlineapps-cloud** [comment]
ok, so on libvirt i able to configure all with success, you planing to add manual mode in future?
![image](https://github.com/user-attachments/assets/3671a070-0290-4713-844d-9b4531a3f47e)

**Comment by AkechiShiro** [comment]
Is this issue solved @onlineapps-cloud ?

Or can you specify it better, you're asking for `manual` support at the end I think, maybe we can close this issue and open a new one tagging as `enhancement` for adding `manual` support.

What do you think @onlineapps-cloud  ?

**Comment by FixeQyt** [comment]
@onlineapps-cloud I know it was closed but switching to X11 worked for me. #8

**Comment by proexposervice** [comment]
<img width="1407" height="499" alt="Image" src="https://github.com/user-attachments/assets/2c5d029c-4fca-41f1-9285-87f1618c4dfc" />
I'm facing the same issue trying to connect to remote machine over network. ./setup.sh completed successfully on manual, but now the ./WinApps-Launcher fails. Is there any other way to launch winapps over network. @onlineapps-cloud ( I know it was closed but switching to X11 worked for me.) - how did you solve it exactly. Can we add manual support for launcher

**Comment by KernelGhost** [comment]
@proexposervice Do the changes merged in [PR #21](https://github.com/winapps-org/winapps-launcher/pull/21) suit your needs?
